src/renderers/common/Backend.js

```diff
@@ -534,9 +534,10 @@ class Backend {
 	/**
 	 * Returns the drawing buffer size.
 	 *
+	 * @param {Vector2} target - The method writes the result in this target object.
 	 * @return {Vector2} The drawing buffer size.
 	 */
-	getDrawingBufferSize() {
+	getDrawingBufferSize( /* target */ ) {
```

Comment:
This line isn't correct. `Backend.getDrawingBufferSize()` has indeed no target.

Reply:
The `target` argument can be removed from below lines:

https://github.com/mrdoob/three.js/blob/297385a377c7f592db7ed07f11f97a442386e295/src/renderers/webgl-fallback/WebGLBackend.js#L458
https://github.com/mrdoob/three.js/blob/297385a377c7f592db7ed07f11f97a442386e295/src/renderers/webgl-fallback/WebGLBackend.js#L658